Documentation/guides/building-apps/build-properties.md

```diff
@@ -1118,8 +1126,13 @@ to continue with dex compilation even if certain warnings are
 encountered. The default value is `True`, but can be set to `False` to
 enforce more strict behavior. See the [ProGuard manual](https://www.guardsquare.com/manual/configuration/usage) for details.
 
+Starting in .NET 8, specifies `--map-diagnostics warning info`. See
+the [D8 and R8 readme][r8-source] for details.
+
 Added in Xamarin.Android 10.3.
 
+[r8-source]: https://r8.googlesource.com/r8/
```

Comment:
Is this the correct link for "details" on `--map-diagnostics`? I do not see anything mentioning `--map-diagnostics`, or even a complete list of supported command-line arguments, at this URL.

Reply:
No, I can't find anything about it, except reading source code and the `--help` output...

Reply:
Then a link to the source code with the `--help` output may be better than a link to the "root" of r8.